runtime/lua/tohtml.lua

```diff
@@ -57,6 +57,26 @@
 --- @field [3] any[][] virt_text
 --- @field [4] any[][] overlay_text
 
+--- @type string[]
+local notifications = {}
+
+---@param msg string
+local function notify(msg)
+  if #notifications == 0 then
+    vim.schedule(function()
+      if #notifications > 1 then
+        vim.notify(
+          ('TOhtml: %s (+ %d more warnings)'):format(notifications[1], tostring(#notifications - 1))
```

Comment:
Maybe vim.notify should do this by default. Eventually `:echo` and all messages will be reworked to behave something like this (with sufficient affordances for seeing the full message).